python/ray/_private/utils.py

```diff
@@ -2001,3 +2001,18 @@ def validate_actor_state_name(actor_state_name):
             'it must be one of the following: "DEPENDENCIES_UNREADY", '
             '"PENDING_CREATION", "ALIVE", "RESTARTING", or "DEAD"'
         )
+
+
+def get_current_node_cpu_model_name() -> Optional[str]:
+    if not sys.platform.startswith("linux"):
+        return None
+
+    try:
+        with open("/proc/cpuinfo", "r") as f:
+            for line in f:
+                if line.startswith("model name"):
```

Comment:
is this guaranteed format? Can you also comment how the string is supposed to look like?

Reply:
Yes, this is guaranteed.